examples/cli/src/main/java/io/unitycatalog/cli/utils/CliParams.java

```diff
@@ -4,6 +4,9 @@ public enum CliParams {
   NAME("name", "The name of the entity.", "name"),
   SCHEMA_NAME("schema", "The name of the schema.", "schema_name"),
   CATALOG_NAME("catalog", "The name of the catalog.", "catalog_name"),
+  MODEL_NAME("model_name", "The name of the registered model.", "model_name"),
```

Comment:
The existing API uses model_name.  We could just translate it to name i guess at the server layer (not sure how confusing that will be to users though).

Reply:
So looking at this further, we can do that mapping between name and model name in the server layer, but this will still not allow us to use the PagedListingHelper since it uses the name as the page token and the name of all model versions under a registered model would be the same.

Reply:
I wouldn't recommend taking this to the server side .. the wiring should happen on the CLI side itself while making the request to the server.

Reply:
I see.  Let me see about updating the CLI to do the mapping then.